public/src/news.js

```diff
@@ -0,0 +1,59 @@
+const app = document.getElementById('root')
+const newlist = document.querySelector('.new-list')
+
+const logo = document.createElement('img')
+logo.src='images/logo.png'
+
+const container = document.createElement('div')
+container.setAttribute('class', 'container')
+
+
+app.appendChild(logo)
+app.appendChild(container)
+
+
+fetch('https://content.guardianapis.com/search?show-blocks=all&api-key=3d1bd118-3d6f-4211-a61e-3d0f966b08bb')
+
+.then(response => {
+  return response.json()
+  
+})
+.then(data => {
+    data.response.results.forEach(news => {
+     var li = document.createElement('li')
+     var a = document.createElement('a')
+     a.setAttribute('href', onclick=retrieve)
+     a.setAttribute('target', '_blank')
+     a.text = "see article"
+     const card = document.createElement('div')
```

Comment:
This all code to create elements should probably be extracted to its own method. Maybe something called `createCard(news)` ?

Reply:
editted.